src/components/Landingpage.js

```diff
@@ -44,6 +46,7 @@ function Landingpage() {
           Ich bin super gespannt auf deine Erlebnisse. Klicke auf den Button und
           fange an mir von deinen Abenteuern zu erzählen.
         </StyledContent>
+        <Button to="/overview" />
```

Comment:
This makes absolut no senese... please remove `to="/overview"`

Reply:
Left that by testing...